.swift-version

```diff
@@ -1 +1 @@
-4.0
+4.1
```

Comment:
Seems like this one is used purely for CocoaPods (don't see any SwiftEnv code) and since CP version 1.4, there is a new variable used instead of deprecated `.swift-version` file. [See this lines](https://github.com/Moya/Moya/blob/f6d736ea6da7e67d1c219980fe10630699a813d1/Moya.podspec#L22-L23) as I've did a similar PR for Moya lately, imho there is 1.52 version of CocoaPods already so we could make the limit on CP >= 1.4.